server/.dockerignore

```diff
@@ -0,0 +1,14 @@
+**/.dockerignore

```

Comment:
Find a better way to do this, instead of ignore a billion folders, just include the ones you want